util/wait/wait.go

```diff
@@ -0,0 +1,52 @@
+package wait
+
+import (
+	"errors"
+	"time"
+)
+
+// Backoff contains the configuration for exponential backoff
+type Backoff struct {
```

Comment:
The asset manager already uses a simple library for exponential backoff `cenkalti/backoff`. Maybe it can be removed and this package used in it's stead.